cmd/admin/admin.go

```diff
@@ -94,3 +95,23 @@ func newAdmin(configFile string, dryRun bool) (*admin, error) {
 		log:    logger,
 	}, nil
 }
+
+// findActiveInputMethodFlag returns the single input method flag that was set
+// to a non-default value in setInputs map. If no flags or multiple flags are
+// set, it returns an error.
+func findActiveInputMethodFlag(setInputs map[string]bool) (string, error) {
+	activeFlags := make([]string, 0, len(setInputs))
```

Comment:
```suggestion
	var activeFlags []string
```

Giving a size hint here is a premature optimization, and gives the impression that `activeFlags` will typically have length `len(setInputs)` by the end of the loop, which is not true in most common cases.

Reply:
Thanks for the feedback! Updated in commit 17c024cab